packages/commonwealth/client/scripts/lib/search_results_cache.ts

```diff
@@ -0,0 +1,43 @@
+type SearchResultsCacheEntry = {
```

Comment:
Can we move this file into helpers/ or elsewhere? the lib/ folder is for external libraries we copy into the repo.